minishift/minishift
# Update Che minishift addon

Fixes N/A

This PR brings the following changes to Che minishift addon:
1. Align labels with che operator provided.
2. Sync Che configuration with upstream up.
3. Use Kubernetes Deployment instead of OpenShift specific DeploymentConfig.
4. Stick Che to 7.0.0 version to avoid dockerimage and configuration mismatching. (7.1.0 would be better but it's not released it, will update version in this PR later, or will create another one if this PR will be already merged after 7.1.0 release of Che)

Steps to test the pull request

  1. Apply minishift-addon
  2. Make sure that Che is started successfully
